src/features/Home/About/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Button, Heading, Text } from '@deriv-com/quill-ui';
+import styles from '../styles.module.scss';
+import Translate from '@docusaurus/Translate';
+import useDeviceType from '@site/src/hooks/useDeviceType';
+
+const About = () => {
+  const { deviceType } = useDeviceType();
+  const isMobile = deviceType === 'mobile';
+
+  return (
+    <div data-testid='about_section' className={styles.aboutContainer}>
+      <div
+        className={`container ${styles.spaceEvenContent} ${styles.aboutContent} ${styles.gap4} ${styles.mh5}`}
```

Comment:
import classNames from 'classnames';
export const joinClassNames = (...classes) => classNames(...classes);
export const classNames = joinClassNames(
  "container",
  styles.spaceEvenContent,
  styles.aboutContent,
  styles.gap4,
  styles.mh5
);

Can we do something like this in a common file and reuse it? This would make it easier to use and allow it to be used in multiple files.

Reply:
I don't think join class is required because combination of these classes is not reusable in our cases because each section have different styling. That's why I make properties common and used it based on necessity.